application/common/helpers/Utils.php

```diff
@@ -138,9 +139,15 @@ public static function maskPhone($phone)
     /**
      * @param string $email an email address (it doesn't verify it)
```

Comment:
This comment is no longer correct (since it does verify... or validate?... the email address).